pkg/host/internal/network/network.go

```diff
@@ -95,10 +96,39 @@ func (n *network) TryGetInterfaceName(pciAddr string) string {
 		return name
 	}
 
-	log.Log.V(2).Info("tryGetInterfaceName()", "name", netDevName)
+	log.Log.V(2).Info("TryGetInterfaceName()", "name", netDevName)
 	return netDevName
 }
 
+// TryGetInterfaceName tries to find the SR-IOV virtual interface index base on pci address
+func (n *network) TryGetInterfaceIndex(pciAddr string) int {
```

Comment:
any chance to add UT for this one ?

Reply:
done both for this function and the vfIsReady function

Reply:
ATM we dont use this method in a "Try" manner but rather fail on error (-1 returned)

WDYT about changing this one to: `GetInterfaceIndex(pciAddr string) (int, error)` ?
i think its better checking error obj than checking for -1.

Reply:
sure done :)